spec/controllers/challenge_claims_controller_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'spec_helper'
+
+RSpec.describe ChallengeClaimsController, type: :controller do
+  include LoginMacros
+  include RedirectExpectationHelper
+  let(:user) { create(:user) }
+  let(:signups) { create(:challenge_signup) }
+  let(:collection) { signups.collection }
+  let(:claim) { create(:challenge_claim, collection: collection) }
+  let(:empty_claim) { create(:challenge_claim) }
+
+  describe 'index' do
+    it 'gives a notice if the collection is closed' do
+      fake_login_known_user(user)
```

Comment:
Since there's a `fake_login_known_user` in every it block, it could probably stand to be moved into a `before` block for the larger context.